src/webgl/p5.Texture.js

```diff
@@ -455,6 +455,15 @@ p5.Texture = class Texture {
   }
 };
 
+// manual cleanup capability
+p5.Texture.prototype.free = function () {
```

Comment:
I think we can just make this method directly in the texture class in this file rather than adding it to the prototype here.